.vib/elasticsearch/goss/vars.yaml

```diff
@@ -0,0 +1,20 @@
+binaries:
+  - elasticsearch
+  - gosu
+  - java
+  - yq
+directories:
+  - paths:
```

Comment:
I think you can also check the permissions for them should be set to 775, as per the following line:

https://github.com/bitnami/containers/blob/53225801ec429d4bae24ea3b2b10458e45267398/bitnami/elasticsearch/8/debian-11/rootfs/opt/bitnami/scripts/elasticsearch/postunpack.sh#L14